src/main/frontend/components/FilmShow.js

```diff
@@ -91,6 +91,7 @@ const FilmShow = props => {
       <ReviewTile
         key={review.id}
         review={review}
+        somethingElse={fetchFilm}
```

Comment:
Also, what is `somethingElse` for? Is that a meaningful property or was it just a placeholder?